src/UI/Component/Input/Field/Factory.php

```diff
@@ -205,4 +206,60 @@ public function dependantGroup(array $inputs);
 	 * @return    \ILIAS\UI\Component\Input\Field\Checkbox
 	 */
 	public function checkbox($label, $byline = null);
+
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     A TagInput is used to choose a subset amount of options out of a finite list
+	 *     of options. The TagInput is used whenever not all available options to choose from
+	 *     MUST or SHOULD be visible, e.g. because the amount is too high (such as all Usernames or a
+	 *     huge amount of Tags).
+	 *     By default, new Options whcih are not yet part of the List of given options, can't be
+	 *     submitted. This can be activated optionally (extendable).
```

Comment:
maybe further clarify with (see withOptionsAreExtendable on the interface) instead of (extendable). Further, maybe also add a use case of when this activation is appropriate, e.g.:
"This is to be used if a list of choices can also be extended with further choices if the option in question is not 
 yet available.".